src/components/organisms/FeaturedAssets.module.css

```diff
@@ -0,0 +1,61 @@
+.container {
+  margin-bottom: calc(var(--spacer) * 2);
+}
+
+.section {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: var(--spacer);
+}
+
+.card {
+  position: relative;
+  composes: box from '../atoms/Box.module.css';
+  width: 100%;
+  height: stretch;
+  overflow: hidden;
+}
+
+.description {
+  width: 65%;
+}
+
+.blob {
+  position: absolute;
+  max-width: 15rem;
+  top: 0;
+  right: 0;
+  pointer-events: none;
+}
+
+.icon {
+  position: absolute;
+  width: 2rem;
```

Comment:
question(non-blocking): Why can we not use the global `--spacer` variable?

thought(non-blocking): it seems like it could make sense refactoring these values to scoped variables. (same for .icon in @media query below)